server/app-engine/static/graas.js

```diff
@@ -85,15 +85,15 @@ function toRadians(degrees) {
 
 // return distance in feet between to lat/long pairs
 function getHaversineDistance(lat1, lon1, lat2, lon2) {
-    const phi1 = toRadians(lat1);
-    const phi2 = toRadians(lat2);
-    const deltaPhi = toRadians(lat2 - lat1);
-    const deltaLam = toRadians(lon2 - lon1);
+    let phi1 = toRadians(lat1);
```

Comment:
We should not replace `const` with `let',  just `var`. `const` has block scope already, which is what we want, plus it enforces immutability.